README.md

```diff
@@ -66,6 +66,14 @@ Following are the implementations managed by third-parties adopting the standard
   * Machine API Operator, https://github.com/openshift/machine-api-operator/tree/master
   * Machine-controller-manager, https://github.com/gardener/machine-controller-manager/tree/cluster-api
 
+## Versioning, Maintenance, and Compatibility
+
+- We follow [Semantic Versioning (semver)](https://semver.org/).
+- Releases/Milestones are set 6 weeks after a Kubernetes release.
+- The cadence is subject to change if necessary, refer to the [Milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) page for up-to-date information.
+- The _master_ branch is where development happens, this might include breaking changes.
+- The _release-X_ branches contain stable, backward compatible code. Every major (X) release, a new such branch is created.
```

Comment:
do we want to come up with a version support scheme now or should we defer until later?

Reply:
I'd defer it until we have a better story around it, wdyt?

Reply:
sgtm